website/go.mod

```diff
@@ -2,4 +2,4 @@ module github.com/cncf/sig-contributor-strategy/website
 
 go 1.15
 
-require github.com/cncf/contribute main // indirect
+require github.com/cncf/contribute v0.0.0-20220713152120-b39b8edc93eb // indirect
```

Comment:
I will update the build scripts/leftover unneeded code around this original solution that pulled in the other repo after this PR is merged.